components/JoinForm/JoinForm.tsx

```diff
@@ -117,6 +117,8 @@ const JoinForm = () => {
             defaultCountry="US"
             value={phoneNumber}
             onChange={setPhoneNumber}/>
+            
+          <input type="text" name="ContactPreference" placeholder="Contact Preference" required />
```

Comment:
This should be a set of radio buttons with two options:
- `Email`
- `Phone`

Email should be the default.